src/web/app/components/loader-bar/loader-bar.component.scss

```diff
@@ -7,7 +7,7 @@
 }
 
 /* stylelint-disable selector-pseudo-element-no-unknown */
-::ng-deep .progress {
+:host ::ng-deep .progress {
```

Comment:
This is not directly related to this PR but I thought I should fix it anyway.
Rationale: If the `::ng-deep` combinator is used without the `:host` pseudo-class selector, the style can bleed into other components. 
Source: https://angular.io/guide/component-styles

Reply:
Any visual of this? Although can't comment much on scss.

Reply:
`:host` scopes the styles to this component, so that it does not affect other components. The styles for this component is unchanged! :)